src/crons/GoG.ts

```diff
@@ -21,20 +21,26 @@ export default new Cron({
   name: 'GoG',
   description:
     'Vérifie toutes les demi heures si GoG offre un jeu (promotion gratuite) et alerte dans #jeux',
-  schedule: '5,35 * * * *',
+  schedule: '5 * * * *',
   // schedule: '* * * * *', // switch for testing
   async handle(context) {
     const game = await getOfferedGame(context.logger);
 
     // vérifie le jeu trouvé avec la dernière entrée
-    const lastGame = await KeyValue.get<string>('Last-Cron-GOG');
-    const gameStoreIdentity = game?.title ?? null;
-    if (lastGame === gameStoreIdentity) return; // skip si identique
+    const gameStoreIdentity = game?.title.normalize().trim() ?? null;
```

Comment:
```suggestion
    const gameStoreIdentity = game?.title;
```

The value is already normalized and supposed to be `Game|null`.